### PR TITLE
do not disable auth for dev and non-prod

### PR DESCRIPTION
**Pull request by rjkerrison**
Let's auth.

### Acceptance Criteria

- [x] master workflow has auth enabled
- [x] pull request workflow has auth enabled
- [x] pull request splats the dev environment

### CODE REVIEW

File: .github/workflows/pars-upload-branch.yaml [hunk 1]
```diff
@@ -79,3 +79,23 @@ jobs:
         with:
           name: medicines-cypress-videos
           path: medicines/pars-upload/cypress/videos
+
+      - name: DEV - Create .env file
+        working-directory: medicines/pars-upload
+        run: |
+          echo NEXT_PUBLIC_CLIENT_ID=26f95b21-63b2-475f-8a35-d39cea4cfd61 >> .env
```
Comment (majikandy) [inline .github/workflows/pars-upload-branch.yaml:86]: let's change this to

```
run: |
          {
            echo "NEXT_PUBLIC_CLIENT_ID=26f95b21-63b2-475f-8a35-d39cea4cfd61"
            echo "NEXT_PUBLIC_AUTHORITY_URL=https://login.microsoftonline.com/e527ea5c-6258-4cd2-a27f-8bd237ec4c26"
            echo "NEXT_PUBLIC_REDIRECT_URI=https://mhraparsdev.azureedge.net"
            echo "NEXT_PUBLIC_DISABLE_AUTH=false"
          } > .env
```

File: .github/workflows/pars-upload-branch.yaml [hunk 2]
```diff
@@ -79,3 +79,23 @@ jobs:
...
+          echo NEXT_PUBLIC_AUTHORITY_URL=https://login.microsoftonline.com/e527ea5c-6258-4cd2-a27f-8bd237ec4c26 >> .env
+          echo NEXT_PUBLIC_REDIRECT_URI=https://mhraparsdev.azureedge.net >> .env
+          echo NEXT_PUBLIC_DISABLE_AUTH=false >> .env
```
Comment (majikandy) [inline .github/workflows/pars-upload-branch.yaml:89]: let's change this to

```
       run: |
          {
            echo "NEXT_PUBLIC_CLIENT_ID=26f95b21-63b2-475f-8a35-d39cea4cfd61"
            echo "NEXT_PUBLIC_AUTHORITY_URL=https://login.microsoftonline.com/e527ea5c-6258-4cd2-a27f-8bd237ec4c26"
            echo "NEXT_PUBLIC_REDIRECT_URI=https://mhraparsdev.azureedge.net"
            echo "NEXT_PUBLIC_DISABLE_AUTH=false"
          } > .env
```

Comment (majikandy) [reply .github/workflows/pars-upload-branch.yaml:89]: and same for the block below for Non prod (but with the non prod values

Comment (rjkerrison) [reply .github/workflows/pars-upload-branch.yaml:89]: it sucks that we're doing two `yarn export`s just because the `REDIRECT_URI` is different :/

**Comment by rjkerrison** [comment]
Calling this blocked until #835 is done because I'm not fixing cypress twice.

File: .github/workflows/pars-upload-branch.yaml (hunk 2 above)
Comment (majikandy) [reply .github/workflows/pars-upload-branch.yaml]: and client_id - but yeah, it does

**Pull request merged**